addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/ForbiddenBypassScanRule.java

```diff
@@ -0,0 +1,193 @@
+        HttpMessage message = getNewMsg();
+        try {
+            sendAndReceive(message);
+            if (message.getResponseHeader().getStatusCode() == HttpStatusCode.FORBIDDEN) {
+                String FULL_URL = message.getRequestHeader().getURI().toString();
+                URI uri = new URI(FULL_URL);
+                String HOST = uri.getAuthority();
+                String PATH = uri.getPath();
+                String SCHEMA = uri.getScheme();
+                List<String> upayloads =
+                        new LinkedList<String>(
+                                Arrays.asList(
+                                        new String[] {
+                                            "/%2e" + PATH,
+                                            PATH + "/.",
+                                            "/." + PATH + "/./",
+                                            PATH + "%20/",
+                                            "/%20" + PATH + "%20/",
+                                            PATH + "..;",
+                                            PATH + "?",
+                                            PATH + "??",
+                                            "/" + PATH + "//",
+                                            PATH + "/",
+                                            PATH + "/.testus",
+                                            PATH + "../app.py"
+                                        }));
+                List<String> hpayloads =
+                        new LinkedList<String>(
+                                Arrays.asList(
+                                        new String[] {
+                                            "X-Rewrite-URL: " + PATH,
+                                            "X-Original-URL: " + PATH,
+                                            "Referer: " + PATH,
+                                            "X-Custom-IP-Authorization: 127.0.0.1",
+                                            "X-Originating-IP: 127.0.0.1",
+                                            "X-Forwarded-For: 127.0.0.1",
+                                            "X-Remote-IP: 127.0.0.1",
+                                            "X-Client-IP: 127.0.0.1",
+                                            "X-Host: 127.0.0.1",
+                                            "X-Forwared-Host: 127.0.0.1"
+                                        }));
+
+                // sending payloads in URL.
+                for (String i : upayloads) {
+                    String TMP_URI = SCHEMA + "://" + HOST + "" + i;
+                    HttpMessage req_with_payload =
+                            new HttpMessage(new org.apache.commons.httpclient.URI(TMP_URI, true));
+                    sendAndReceive(req_with_payload);
+                    if (req_with_payload.getResponseHeader().getStatusCode() == HttpStatusCode.OK) {
+                        newAlert()
+                                .setConfidence(Alert.RISK_INFO)
```

Comment:
Was this meant to be `setRisk`?

Reply:
I think its a type of alert [High, Medium, Low or Info]. I am not completely sure because I lean it from other active scan rules.

Reply:
I mean it's setting/using the constant from the risk instead of the ones for the confidence, as is now is setting the alert to false positive.